ocf_reader.go

```diff
@@ -117,6 +117,21 @@ func (ocfr *OCFReader) Read() (interface{}, error) {
 	return datum, nil
 }
 
+// ReadBinary simply returns the []byte rather than calling NativeFromBinary
+func (ocfr *OCFReader) ReadBinary() ([]byte, error) {
+	// NOTE: Test previous error before testing readReady to prevent overwriting
+	// previous error.
+	if ocfr.rerr != nil {
+		return nil, ocfr.rerr
+	}
+	if !ocfr.readReady {
+		ocfr.rerr = errors.New("Read called without successful Scan")
+		return nil, ocfr.rerr
+	}
+
```

Comment:
Are you intending for this function to be called right after `Read()`?

Reply:
I think this is before the `Read()`....a peek before advancing over in `Read()`

Reply:
I'd have simply added an additional return value of type `[]byte` to `Read()`. This would remove any chance of confusion regarding the order in which these two methods should be called.